src/components/file_picker.vue

```diff
@@ -242,24 +259,207 @@ export default {
     },
     slugs: function() {
       return this.path.split(pathmod.sep);
+    },
+    showFsFavorites: function() {
+      return this.fs_favorites.length > 0;
+    },
+    fsEntryColWidth: function() {
+      return (this.showFsFavorites) ? 'col-sm-7' : 'col-sm-12';
     }
   }
 }
 </script>
 
 <style scoped>
-.glyphicon.spinning {
-    animation: spin 1s infinite linear;
-    -webkit-animation: spin2 1s infinite linear;
-}
+  /* Begin Bulma Modal */
```

Comment:
I initially tried just working with Bootstrap's modal classes just without using their JS. I found it slow going. The CSS framework [Bulma](https://bulma.io/) on the other hand does not provide JS, and relies on/allows the user to determine what JS best meets their needs. What follows is the entire Bulma `.modal` section. Unused parts could be removed, or a more concerted effor to use pure Bootstrap could be made.